apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-content.js

```diff
@@ -22,10 +22,21 @@ function getTourContent( localeSlug ) {
 	return [
 		{
 			heading: __( 'Welcome to WordPress!', 'full-site-editing' ),
-			description: __(
-				'Continue on with this short tour to learn the fundamentals of the WordPress editor.',
-				'full-site-editing'
-			),
+			description:
+				localeSlug === 'en' ||
+				localeSlug === 'en-gb' ||
+				hasTranslation(
+					'Take this short, interactive tour to learn the fundamentals of the WordPress editor.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Continue on with this short tour to learn the fundamentals of the WordPress editor.' )` *ES Score: 7*